### ISSUE TITLE
[docs] link to plugin README template in Creating a Plugin guide

### ISSUE THREAD

**Issue by gillkyle** [gatsbyjs/gatsby#14774]
## Summary

An important hurdle users trying to leverage plugins encounter is not enough documentation, or difficult to understand documentation on plugins. In the [Creating Plugins](https://www.gatsbyjs.org/docs/creating-plugins/) guide there currently isn't much information (only a few sentences at the end of the Submit to Plugin Library section) guiding a plugin author on what should be included in their README. 

There is already [a template for plugin READMEs](https://www.gatsbyjs.org/contributing/docs-templates/#plugin-readme-template) thanks to #4284 that exists but it is not linked to or mentioned in that guide.  

## What to Do

Some helpful things that could be added:

- linking to the [plugin README template](https://www.gatsbyjs.org/contributing/docs-templates/#plugin-readme-template) from the page [Submit to Plugin Library](https://www.gatsbyjs.org/contributing/submit-to-plugin-library/) with a note about the template
- (maybe?) adding a page to the Creating Plugins guide that briefly explains the README template and links to it, perhaps also explaining how good documentation can help new users get started with the plugin

**Comment by k-conway** [comment]
I would like to take this one!

**Comment by gillkyle** [comment]
Thanks @k-conway! It looks like you forked the repository (which basically created your own version of Gatsby on your GitHub profile), sounds like you'll need to do a couple things if you haven't already:

1. set your fork to track this repository (run this in your folder with your Gatsby fork):

```
git remote add upstream https://github.com/gatsbyjs/gatsby.git
```

2. then you'll need to create your own branch and push your changes to it

3. then you should see the option to create a PR in the GitHub UI with the changes you've made

**Comment by gillkyle** [comment]
Hey @k-conway have you been able to get to this? 🙂

**Comment by k-conway** [comment]
hi! I will be back at this today! sorry for the delay!

**Comment by gillkyle** [comment]
@k-conway no worries, feel free to get to it when you have time, just want to make sure you don't get stuck and/or frustrated 👍

**Comment by k-conway** [comment]
Thank you so much for reaching out! I just sent in my Pull Request!

**Comment by gillkyle** [comment]
Thanks @k-conway, merged in #14856
